tests/space/conftest.py

```diff
@@ -24,6 +24,7 @@
 import pytest
 
 from xyzspaces.apis import HubApi
+from xyzspaces.config.default_config import XYZConfig
 from xyzspaces.datasets import get_chicago_parks_data, get_countries_data
 from xyzspaces.spaces import Space
 from xyzspaces.utils import get_xyz_token
```

Comment:
Is this still needed, then, here and elsewhere?

Reply:
`get_xyz_token` is not needed here in tests now, but it is a utility function so user can use it if needs to get token from env.